script/travis-script

```diff
@@ -16,7 +16,7 @@ mix compile
 
 # report test coverage to coveralls.io
 # shellcheck disable=SC2037
-MIX_ENV=test mix coveralls.travis
+MIX_ENV=test mix coveralls.travis --exclude slow
 
 # report test coverage to codecov
 # shellcheck disable=SC2037
```

Comment:
does the next line require modification as well?

Reply:
The reason why this line required modification is because, it was running all the tests, and as we discussed, we want to exclude the slow tests.